lib/presentation/views/web_home_view.dart

```diff
@@ -1,22 +1,97 @@
 
 class WebLayout extends StatelessWidget {
   final List<Article> articles;
-  const WebLayout({required this.articles, super.key});
+  final double size;
+  final String username;
+  const WebLayout(
+      {required this.articles,
+      required this.size,
+      required this.username,
+      super.key});
 
   @override
   Widget build(BuildContext context) {
-    return GridView.count(
-      crossAxisCount: 3,
-      children: List.generate(articles.length, (index) {
-        return HomePageArticle(
-          article: articles[index],
-          size: MediaQuery.of(context).size.width * 0.35,
-        );
-      }),
+    return Row(
+      children: [
+        Container(
+          width: size * 0.7,
+          child: Column(children: [
+            Padding(
+              padding: EdgeInsets.all(size * 0.05),
+              child: Row(
+                children: [
+                  CircleAvatar(
+                    backgroundImage:
+                        const AssetImage('assets/images/profile.png'),
+                    radius: size * 0.07,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: size * 0.035),
+                    child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Text(
+                            'Welcome',
+                            style: TextStyle(
+                                color: Colors.grey, fontSize: size * 0.025),
+                          ),
+                          Text(
+                            username,
+                            style: TextStyle(
+                                color: Colors.black, fontSize: size * 0.045),
+                          ),
+                        ]),
+                  ),
+                  const ImageIcon(
+                    AssetImage('assets/images/arrow.png'),
+                  )
+                ],
+              ),
+            ),
+            DrawerTile(
+              title: 'Explore',
+              icon: ImageIcon(AssetImage('assets/images/explore.png')),
+              size: size,
+            ),
+            DrawerTile(
+              title: 'Live Chat',
+              icon: ImageIcon(AssetImage('assets/images/live.png')),
+              size: size,
+            ),
+            DrawerTile(
+              title: 'Gallery',
+              icon: ImageIcon(AssetImage('assets/images/gallery.png')),
+              size: size,
+            ),
+            DrawerTile(
+              title: 'Whish List',
+              icon: ImageIcon(AssetImage('assets/images/wishlist.png')),
+              size: size,
+            ),
+            DrawerTile(
+              title: 'E-Magazine',
+              icon: ImageIcon(AssetImage('assets/images/e-magazine.png')),
+              size: size,
+            ),
+          ]),
+        ),
```

Comment:
it's good practice to use the same ui for all as it's the same (Mobile, landscape and web) and take care of using column when the height can be less than the data height

<img width="1307" alt="Screenshot 2024-07-23 at 10 45 13 AM" src="https://github.com/user-attachments/assets/331abe5b-8152-441b-bd7c-a7738ab5d04e">